chapters/annotations.tex

```diff
@@ -163,10 +163,11 @@ \subsection{Class Description and Revision History}\label{annotation-info-revisi
 The \lstinline!revisions! documentation may be omitted in printed documentation.
 \end{nonnormative}
 
-If the string starts with the tag \lstinline!<html>! or \lstinline!<HTML>! the entire string is HTML encoded (and is assumed to end with \lstinline!</html>! or \lstinline!</HTML>! and shall be rendered as HTML even if the end-tags are missing), otherwise the entire string is rendered as is.
-The HTML encoded content may contain links.
+If the string starts with the tag \lstinline!<html>! or \lstinline!<HTML>!, the entire string is HTML encoded (assumed to end with \lstinline!</html>! or \lstinline!</HTML>!, but to be rendered as HTML even if the end-tag is missing).
+Otherwise, the entire string is rendered as is.
+HTML encoded content may contain links.
 For external links, see \cref{external-resources}.
-Links to Modelica classes may be defined with the HTML link command using scheme \lstinline!Modelica! (using its lower case form in the URI, see \cref{external-resources}), e.g.,
+The Modelica-scheme may be used to refer to Modelica classes, e.g.,
```

Comment:
Should it be `modelica-scheme` (lowercase, like the URI uses)?

Reply:
No, I think there is a deliberate choice behind the case used in `\cref{external-resources}`.

Reply:
OK now?